README.md

```diff
@@ -19,11 +19,12 @@ Load the credentials into a `GitCredential` object, and pick a folder where to s
 ```python
 from kfcicli.main import *
 import json
+import os
 
 with open("credentials.json", "r") as fid:
     credentials = GitCredentials(**json.loads(fid.read()))
 
-tmp_folder = "/home/deusebio/tmp/kfcicli/charm_repos"
+tmp_folder = f"/home/os.getenv('HOME')/tmp/kfcicli/charm_repos"
```

Comment:
`USER` or `HOME`?

If you use HOME, we should drop the "/home" at the beginning